### PR TITLE
Added Redis and Deno tutorial

### PR DESCRIPTION
**Pull request by ajeetraina**

### CODE REVIEW

File: docs/develop/deno/index.deno.mdx [hunk 1]
```diff
@@ -0,0 +1,114 @@
+---
+id: index-deno
+title: Deno and Redis
+sidebar_label: Deno 
+slug: /develop/deno/
+---
+
+
+[With over 80,000 stars and 670+ contributors](https://github.com/denoland/deno), Deno is a popular modern runtime for JavaScript and TypeScript. It is built on [V8](https://v8.dev/) - an open-source JavaScript engine developed by the Chromium Project for Google Chrome and Chromium web browsers. Companies like GitHub, Amazon, Begin, IBM, Netlify, Siemens, Vercel etc.
```
Comment (simonprickett) [inline docs/develop/deno/index.deno.mdx]: @ajeetraina shoud the last sentence "Companies like GHitHub..." say something like "contribute to Deno development" or "use Deno" or something?  It feels like it's missing something right now.

File: docs/develop/deno/index.deno.mdx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+### Features of Deno
+
+
+- Deno is secure by default. It executes the code in a sandbox environment, disallowing runtime to the underlying filesystem, environmental variables and scripts.- Deno supports both JavaScript and  TypeScript out of the box.
```
Comment (simonprickett) [inline docs/develop/deno/index.deno.mdx]:
```suggestion
- Deno is secure by default. It executes code in a sandbox environment, disallowing runtime access to the underlying filesystem, environment variables and scripts.  
- Deno supports both JavaScript and TypeScript out of the box.
```

Comment (simonprickett) [inline docs/develop/deno/index.deno.mdx]:
```suggestion
- Deno is secure by default. It executes code in a sandbox environment, disallowing runtime access to the underlying filesystem, environment variables and scripts.  Deno supports both JavaScript and TypeScript out of the box.
```

File: docs/develop/deno/index.deno.mdx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+- Deno ships as a single executable with no dependencies.
+- Comes with built-in utilities like a dependency inspector (deno info) and a code formatter (deno fmt).
+
+
+### Getting Started
+
+[deno.land/x](https://deno.land/x) is a hosting service for Deno scripts. It caches releases of open source modules stored on GitHub and serves them at one easy to remember domain.These module contains small scripts that demonstrate use of Deno and its standard module.
```
Comment (simonprickett) [inline docs/develop/deno/index.deno.mdx]:
```suggestion
[deno.land/x](https://deno.land/x) is a hosting service for Deno scripts. It caches releases of open source modules stored on GitHub and serves them at one easy to remember domain. These modules contain small scripts that demonstrate use of Deno and its standard module.
```

File: docs/develop/deno/index.deno.mdx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+```
+
+#### Example: 
+ 
+```
+https://deno.land/std@0.126.0/examples
+```
+
+
+In order to use Redis with Deno you will need a Deno Redis client. In the following sections, we will demonstrate the use of [an experimental implementation of Redis client for Deno](https://deno.land/x/redis@v0.25.3).
```
Comment (simonprickett) [inline docs/develop/deno/index.deno.mdx]:
```suggestion
In order to use Redis with Deno you will need a Deno Redis client. In the following sections, we will demonstrate the use of [an experimental implementation of a Redis client for Deno](https://deno.land/x/redis@v0.25.3). 
```

File: docs/develop/deno/index.deno.mdx [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+### Step 2. Run Redis locally
+ 
+```
+docker run -d -p 6379:6379 redislabs/redis
+```
+
+### Step 3. Create an empty file with the following content
+
+
+The following code creates a connection to Redis using Deno
```
Comment (simonprickett) [inline docs/develop/deno/index.deno.mdx]:
```suggestion
The following code creates a connection to Redis using Deno:
```

File: docs/develop/deno/index.deno.mdx [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+import { connect } from "https://deno.land/x/redis/mod.ts";
+const redis = await connect({
+  hostname: "127.0.0.1",
+  port: 6379,
+});
+const ok = await redis.set("foo", "bar");
+const foo = await redis.get("foo");
+```
+
+Replace the following values with those of your database and save this file as redis.ts
```
Comment (simonprickett) [inline docs/develop/deno/index.deno.mdx]: "Replace the following values" -> this doesn't currently go on to state what values those are  - perhaps change to "replace the values of `hostname` and `port` to match those of your Redis database, and add an extra `password` field if needed.

File: docs/develop/deno/index.deno.mdx [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+
+### Step 4. Executing the script
+
+Deno can grab the scripts from multiple sources, a filename, a url, and '-' to read the file from stdin.
```
Comment (simonprickett) [inline docs/develop/deno/index.deno.mdx]:
```suggestion
Deno can grab scripts from multiple sources.  For example, you can provide a filename, a url, or'-' to read the file from stdin. 
```

File: docs/develop/deno/index.deno.mdx [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+const ok = await redis.set("foo", "bar");
+const foo = await redis.get("foo");
+```
+
+Replace the following values with those of your database and save this file as redis.ts
+
+### Step 4. Executing the script
+
+Deno can grab the scripts from multiple sources, a filename, a url, and '-' to read the file from stdin. 
+You can run a JavaScript or TypeScript program by executing the ```deno run```.
```
Comment (simonprickett) [inline docs/develop/deno/index.deno.mdx]:
```suggestion
You can run a JavaScript or TypeScript program by executing ```deno run```.
```

File: docs/develop/deno/index.deno.mdx [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+
+
+```
+deno run --allow-net redis.ts
+
+```
+
+It should display the value of foo output. You can verify this by running the monitor command
```
Comment (simonprickett) [inline docs/develop/deno/index.deno.mdx]:
```suggestion
When you run the script, the value of `foo` should be output. You can verify this by running the monitor command:
```

File: docs/develop/deno/index.deno.mdx [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+monitor
+OK
+1645524375.012602 [0 172.17.0.1:63288] "SET" "foo" "bar"
+1645524375.020807 [0 172.17.0.1:63288] "GET" "foo"
+
+```
+
+
+
+### Further Reference:
```
Comment (simonprickett) [inline docs/develop/deno/index.deno.mdx]:
```suggestion
### Further References:
```

File: docs/develop/deno/index.deno.mdx [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+
+- [Getting Started with Deno Commandline Interface](https://deno.land/manual@v1.19.0/getting_started/command_line_interface)
```
Comment (simonprickett) [inline docs/develop/deno/index.deno.mdx]:
```suggestion
- [Getting Started with Deno Command Line Interface](https://deno.land/manual@v1.19.0/getting_started/command_line_interface)
```

**Review by simonprickett — APPROVED** [review]
@ajeetraina left a couple of comments to look at - approved.

File: docs/develop/deno/index.deno.mdx (hunk 1 above)
Comment (ajeetraina) [reply docs/develop/deno/index.deno.mdx]: Oh yes...its missing text. I will fix it.

Comment (ajeetraina) [reply docs/develop/deno/index.deno.mdx]: I will remove this statement instead.

**Pull request merged**